ossdbs/fem/volume_conductor/volume_conductor_model.py

```diff
@@ -257,7 +260,11 @@ def run_full_analysis(
                     # currently: refine until impedance doesn't change by more than 0.1%
                     error = 100
                     refinements = 0
-                    while error > 0.1 and refinements < 10:
+                    # TODO write a meaningful algo
+                    while (
+                        error > 0.1
```

Comment:
Make the error also a parameter in the dictionary

Reply:
Done